packages/changed-elements-react/src/api/VersionCompareVisualization.ts

```diff
@@ -60,6 +58,7 @@ export class VersionCompareVisualizationManager {
   private _focusedElements: ChangedElementEntry[] | undefined;
 
   private _modelsAtStart: string[] = [];
+  private _EEBeforeVcRun: EmphasizeElementsProps | undefined;
```

Comment:
Rename to follow camel case: `_eeBeforeVcRun`, maybe even better name: `_originalEmphasizeElementsProps`